drivers/tee/optee/call.c

```diff
@@ -220,6 +220,12 @@ int optee_open_session(struct tee_context *ctx,
 	struct optee_msg_arg *msg_arg;
 	phys_addr_t msg_parg;
 	struct optee_session *sess = NULL;
+	uuid_t client_uuid;
+
+	rc = tee_session_calc_client_uuid(&client_uuid, arg->clnt_login,
```

Comment:
This call can be moved to replace the `memcpy()` below and saving us from keeping `client_uuid` on the stack.
The reason I care is the call to `optee_do_call_with_arg()` which may do some rather deep function calls during RPC handling so `client_uuid` will directly add to the maximum amount of stack used. Since this saving doesn't cost anything I can see no reason to not do it.

Reply:
ok